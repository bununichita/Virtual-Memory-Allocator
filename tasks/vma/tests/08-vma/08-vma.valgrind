==36858== Memcheck, a memory error detector
==36858== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==36858== Using Valgrind-3.18.1 and LibVEX; rerun with -h for copyright info
==36858== Command: /home/nicky/Desktop/SDA/Virtual-Memory-Allocator/vma
==36858== Parent PID: 36857
==36858== 
==36858== 
==36858== HEAP SUMMARY:
==36858==     in use at exit: 0 bytes in 0 blocks
==36858==   total heap usage: 10 allocs, 10 frees, 8,377 bytes allocated
==36858== 
==36858== All heap blocks were freed -- no leaks are possible
==36858== 
==36858== For lists of detected and suppressed errors, rerun with: -s
==36858== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
