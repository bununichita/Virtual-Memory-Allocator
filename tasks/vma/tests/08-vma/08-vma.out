The end address is past the size of the arena
This zone was already allocated.
Total memory: A bytes
Free memory: 9 bytes
Number of allocated blocks: 1
Number of allocated miniblocks: 1

Block 1 begin
Zone: 5 - 6
Miniblock 1:	5	-	6	| RW-
Block 1 end
1
5
